index.md

```diff
@@ -1492,6 +1492,32 @@ Reply Examples:
       :ircd.stealth.net 302 yournick :syrk=+syrk@millennium.stealth.net
                                       ; Reply for user syrk
 
+### WALLOPS message
+
+         Command: WALLOPS
+      Parameters: <Text to be sent>
```

Comment:
"Text to be sent" is a good description for the WALLOPS client-to-server message, but not so much for the server-to-client message. Maybe consider just using "text" just like in e.g. AWAY?